resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/ezbake.spec.erb

```diff
@@ -262,7 +265,7 @@ fi
 %{_sym_bindir}/<%= bin_file.sub(%r{^ext/bin/}, "") %>
 %{_ux_bindir}/<%= bin_file.sub(%r{^ext/bin/}, "") %>
 <% end -%>
-%dir %attr(0770, <%= EZBake::Config[:user] %>, <%= EZBake::Config[:group] %>) %{_app_data}
+%attr(0770, <%= EZBake::Config[:user] %>, <%= EZBake::Config[:group] %>) %{_app_data}
```

Comment:
Updated to set the dir perms correctly when we install, and not touch the files here